### PR TITLE
Integrate Firebase

### PR DESCRIPTION
**Pull request by jpirazusta**
### Pull request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Config

---

### Description

Integrated Firebase for the different environments for android and iOS.

### CODE REVIEW

**Review by ManuViola77 — APPROVED** [review]
👏🏻

File: android/app/src/develop/google-services.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (marcos09) [inline android/app/src/develop/google-services.json:1]: I think you added this file before adding the path to the gitignore. Could you remove it?

**Review by marcos09 — CHANGES_REQUESTED** [review]
I think you added the google-services key. Please remove it, and I'll approve the PR.

File: android/app/src/develop/google-services.json (hunk 1 above)
Comment (jpirazusta) [reply android/app/src/develop/google-services.json]: I don't know why the development one is appearing, since all are in gitignore. I removed it

**Review by marcos09 — APPROVED** [review]

**Pull request merged**